<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transprent;
</string>
  </property>
  <widget class="QLabel" name="label_banner">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #fff;}</string>
   </property>
   <property name="text">
    <string>Zip Extractor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zip_file_path">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { 
	border: 2px solid rgb(94, 94, 94);
	border-radius: 5px;
	color: #fff;
}
QLineEdit:hover {
	border-color: #fff;
}</string>
   </property>
   <property name="placeholderText">
    <string>Zip File Path...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="extract_file_path">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { 
	border: 2px solid rgb(94, 94, 94);
	border-radius: 5px;
	color: #fff;
}
QLineEdit:hover {
	border-color: #fff;
}</string>
   </property>
   <property name="placeholderText">
    <string>Extract File Path...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #fff;
}</string>
   </property>
   <property name="text">
    <string>Zip File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #fff;
}</string>
   </property>
   <property name="text">
    <string>Extract Path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_file">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #fff;
	border-radius: 5px;
}
QPushButton:hover {
	border: 2px solid gray;
	background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>Choose File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_file_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #fff;
	border-radius: 5px;
}
QPushButton:hover {
	border: 2px solid gray;
	background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>Choose File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="extract_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #90ee90;
	border: 2px solid #90ee90;
	border-radius: 5px;
}
QPushButton:hover {
	background-color: #90ee90;
	color: #000;
}</string>
   </property>
   <property name="text">
    <string>Extract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #90ee90;
	border: 2px solid #90ee90;
	border-radius: 5px;
}
QPushButton:hover {
	background-color: #90ee90;
	color: #000;
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_v">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color: #fff; }</string>
   </property>
   <property name="text">
    <string>File Extractor v1.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
